src/gdswzd_c.F90

```diff
@@ -226,7 +227,10 @@ SUBROUTINE GDSWZD_C(IGDTNUM,IGDTMPL,IGDTLEN,IOPT,NPTS,FILL, &
  REAL(KIND=C_FLOAT), INTENT(OUT)          :: YLON(NPTS),YLAT(NPTS),AREA(NPTS)
 #endif
 
```

Comment:
I often call ip and ip2 only for the gdswzd routine (no interpolation).  In that case, I can declare a 'type(grib1/grib2_descriptor)' variable in my program, then call gdswzd, correct?  That is a public function?

Reply:
I left the external interface the same. It converts to a descriptor internally.